pyvisa-py/gpib.py

```diff
@@ -189,6 +189,50 @@ def clear(self):
         except:
             return 0, StatusCode.error_system_error
 
+    def gpib_command(self, command_byte):
+        """Write GPIB command byte on the bus.
+
+        Corresponds to viGpibCommand function of the VISA library.
+        See: https://linux-gpib.sourceforge.io/doc_html/gpib-protocol.html#REFERENCE-COMMAND-BYTES
+
+        :param command_byte: command byte to send
+        :type command_byte: int, must be [0 255]
+        :return: return value of the library call
+        :rtype: :class:`pyvisa.constants.StatusCode`
+        """
+
+        if 0 <= command_byte <= 255:
+            data = chr(command_byte)
+        else:
+            return StatusCode.error_nonsupported_operation
+
+        try:
+            self.controller.command(data)
+            return StatusCode.success
+
+        except gpib.GpibError:
+            # err = self.interface.ibsta()
```

Comment:
Why is this commented out ? Should it be removed ?

Reply:
Yeah. I'll get rid of it.